lib/getNetlifyRoute.js

```diff
@@ -9,7 +9,7 @@
 // also handles catch all routes /[...param]/ -> /:*
 module.exports = dynamicRoute => {
   // replace any catch all group first
-  const expressified = dynamicRoute.replace(/\[\.\.\.(.*)]$/, "*");
+  const expressified = dynamicRoute.replace(/\[{1,2}\.{3}(.*)]{1,2}$/, "*");
```

Comment:
```suggestion
  const expressified = dynamicRoute.replace(/\[+\.+(.*)]+$/, "*");
```
could work also for arbitrary `[`, `.`, `]`s if that's clearer, but current is more accurate